src/main/java/com/jalasoft/selenium/daniel/jauregui/movies/Customer.java

```diff
@@ -1,20 +1,23 @@
 package com.jalasoft.selenium.daniel.jauregui.movies;
 
 import java.util.ArrayList;
-import java.util.Iterator;
+
 /**
  * Customer class.
  * @author Daniel Jauregui
  */
 class Customer {
     private final String name;
     private final ArrayList<Rental> rentals = new ArrayList<Rental>();
```

Comment:
It is not a good practice to instantiate a class in the attributes.
Please move it to the constructor.